rmf_schedule_visualizer/src/VisualizerData.cpp

```diff
@@ -102,6 +102,32 @@ void VisualizerDataNode::start(Data _data)
       debug_cb(std::move(msg));
     });
 
+  // retrieve/construct mirrors, snapshots and negotiation object
+  {
+    _writer = rmf_traffic_ros2::schedule::Writer::make(*this);
```

Comment:
As mentioned in another comment, this `_writer` is unused, so we should simply remove it.

Reply:
removed 3b1c580